src/backend/backend.ts

```diff
@@ -174,6 +176,9 @@ export interface VMBackend {
    * If true, the backend cannot invoke any dialog boxes and needs to find an alternative.
    */
   noModalDialogs: boolean;
+
+  readonly executor: VMExecutor;
+  readonly kube: KubernetesBackend;
```

Comment:
I think this is the wrong name for this object.
`kube` is kind of generic when talking about Kubernetes, and it's not a logical abbrev'n for `KubernetesBackend` (`kuba` isn't so great either). I would prefer `kbackend` or `kBackend` or `kubeBackend` or even `kubernetesBackend` -- in increasing order of preference. I see it's (currently) only used in `background.ts` and only 17 times so a longer name isn't that onerous.

Reply:
Changed to `kubeBackend` in 580f3b941d62e5d422f297b784b90e62f4f91151.

I originally called it `kube` on the assumption that the callers would end up looking like `backend.kube.method(…)`, to avoid stutter. (It's currently called `k8smanager` in most places, but that will need to be updated anyway.)